### PR TITLE
Initial Mongodb support

### PR DESCRIPTION
**Pull request by robbyboparai**

### CODE REVIEW

File: sturdycouscous/Garbanzo/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+cryptography==3.2.1
+constantly==15.1.0
+chardet==3.0.4
+cffi==1.14.4
+certifi==2020.11.8
+beautifulsoup4==4.9.3
+Automat==20.2.0
+attrs==20.3.0
+async-timeout==3.0.1
+aioredis==1.3.1
```
Comment (IsabelleLenertz) [inline sturdycouscous/Garbanzo/requirements.txt]: I there I thought I was the one adding too many external dependencies ><

File: sturdycouscous/run_db_test.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	docker network create $NETWORK_NAME 
+}
+
+function launchMongo() {
+	docker build -t dbtestimage $PWD/db/.
+	docker run --network=$NETWORK_NAME --name $MONGO_CONTAINER_NAME -e MONGO_INITDB_ROOT_USERNAME=root -e MONGO_INITDB_ROOT_PASSWORD=root -h $MONGO_CONTAINER_NAME dbtestimage &
+}
+
+function launchCore() {
+	docker build -t sturdy-couscous $PWD/Garbanzo/.
```
Comment (janani-sridhar) [inline sturdycouscous/run_db_test.sh]: should this be changed to $PWD/. ?

File: sturdycouscous/Garbanzo/requirements.txt (hunk 1 above)
Comment (janani-sridhar) [reply sturdycouscous/Garbanzo/requirements.txt]: should requirements.txt be a directory up?

**Review by janani-sridhar — CHANGES_REQUESTED** [review]
I thought we had discussed that we're not gonna be building mongo based off of Garbanzo, but based off of the sturdycouscous directory itself?

File: sturdycouscous/Garbanzo/requirements.txt (hunk 1 above)
Comment (robbyboparai) [reply sturdycouscous/Garbanzo/requirements.txt]: It was a short list before I did a `pip freeze` to pin the versions of the original 7 packages: https://github.com/IsabelleLenertz/sturdy-couscous/pull/20/commits/79908e587239d0138e5ec71791272c4ba96fc16d
And yeah, I can move it one directory up, no problem

**Pull request closed without merging**